### DISCUSSION TITLE
Project logo?

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#667] [Ideas]
While playing with Batik I came up with something that could be potentially used as project logo. From what I can see, there's nothing like that for this project, so unless someone can make anything more professionally designed, maybe it could do the trick? Looks pretty OK in About dialog :) Thoughts?

![about](https://user-images.githubusercontent.com/8041294/121815332-3c037980-cc76-11eb-81e6-63e189b8edab.png)

**Comment by BFH-ktt1** [comment]
I like it, and am always open to such nice improvements :-)

**Reply by MarcinOrlowski** [reply]
Great. I will make a PR with this particular change once #687 land (as it touches the same part of the code).

**Reply by MarcinOrlowski** [reply]
This is now merged in #700
